docs/cli/reference/feature-flags.md

```diff
@@ -48,21 +48,23 @@ Example configuration file
 
 ```json
 {
-    "features": {
-        "graphQLTransformer": {
-            "transformerVersion": 5
-        },
-        "keyTransformer": {
-            "defaultQuery": true
-        },
-        "experimental": {
-            "feature1": true,
-            "feature2": false
-        },
-        "release": {
-            "lambdaLayers": true
-        }
+  "features": {
+    "graphqltransformer": {
```

Comment:
use camel case in the feature names. makes it easier to read and the CLI normalizes to lower case when reading the file

Reply:
This is how it's stored in the file. The names are converted to lowercase by the CLI. I just copied the current `cli.json` file so that customers know what to expect.